spec/features/admin/add_custom_organisation_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'support/confirmation_use_case'
+
+describe 'adding a custom organisation name' do
+  include_examples 'confirmation use case spy'
+
+  let!(:admin_user) { create(:user, super_admin: true) }
+
+  context 'when visiting the custom organisations page' do
+    before do
+      sign_in_user admin_user
+      visit admin_custom_organisations_path
+    end
+
+    it 'will show the add custom organisations page' do
+      expect(page.body).to have_content('Allow an organisation access to the admin platform')
+    end
+
+    it 'will redirect you to organisations page once a custom org is added' do
+      fill_in "Enter the organisation's full name", with: 'Custom Org name'
+      expect do
+        click_on 'Allow organisation'
+        .to change { CustomOrganisationName.count }.by(1)
+      end
+    end
+
+    context 'sign out and find custom organisations' do
+      before do
+        sign_out
+        CustomOrganisationName.create(name: 'Custom Org name')
+        sign_up_for_account(email: 'default@gov.uk')
+        visit confirmation_email_link
+      end
+
+      it 'displays the custom organisation name in the list` would be better for instance' do
```

Comment:
Haha George. You blindly copied and pasted the entire line

Reply:
👁

Reply:
LOOOOL